minefield_game/board_header.h

```diff
@@ -9,9 +9,7 @@ struct Board
     std::vector<std::vector<Cell>> grid;
 
 };
-//void showBoard(const Board& board);
 namespace BoardUtils

```

Comment:
#should preferably use dromedaryCase instead of CamelCase for namespaces. CamelCase is most widely used for classes and structs, so it'd be easier to confuse with one of those.